public_dropin_gpu_environments/nemollm_inference_ms/artifact_downloader/datarobot_custom_code/model_artifact_downloader.py

```diff
@@ -0,0 +1,542 @@
+import datetime
+import time
+import os
+import pprint
+import json
+import subprocess
+from functools import partial
+from multiprocessing import Lock, Manager, Pool, Process
+
+from datarobot.mlops.connected.client import MLOpsClient
+from datarobot.mlops.mlops import MLOps
+import datarobot as dr
+
+from datarobot_custom_code.utils import bytes_to_mb_str, calculate_rate, calculate_rate_str, parse_s3_uri, list_zip_contents, \
+    extract_zip_content, load_yaml_from_file, sum_file_sizes, get_disk_space, merge_lists
+
+from datarobot_custom_code.s3_file_download_helper import S3FileDownloadHelper
+
+
+class SupportedStoreType:
+    S3 = "s3"
+
+
+class StoreURIPrefix:
+    S3 = "s3:"
+
+
+class ModelDownloader:
+    REMOTE_FILE_SUFFIX = ".remote"
+    METADATA_FILE = "model-metadata.yaml"
+    MLOPS_RUNTIME_PARAM_PREFIX = "MLOPS_RUNTIME_PARAM_"
```

Comment:
We shouldn't do this. If we want to interact with runtime params, we need to use the standard Python abstraction.

Reply:
Maybe we'll skip this library for this release altogether? It's not ready yet.

Reply:
@elatt I'm going to close this PR, I feel that we have to skip this library from the current release and add it when it's covered with tests, have it's own pipeline to be published, etc.

Users have ability to run the `custom.py` load_model hook.

Reply:
> Maybe we'll skip this library for this release altogether? It's not ready yet.

Sounds fine with me 🤷‍♂️